test/functional/wallet_importmulti.py

```diff
@@ -584,12 +584,12 @@ def run_test(self):
         self.test_importmulti({"desc": descsum_create(desc),
                                "timestamp": "now",
                                "range": 1},
-                              success=True,
-                              warnings=["Some private keys are missing, outputs will be considered watchonly. If this is intentional, specify the watchonly flag."])
+                              success=True)
```

Comment:
Comment above (`# Test importing of a ranged descriptor without keys`) needs to be updated to something like `# Test importing of a ranged descriptor with xpriv`